apps/fulltextsearch.sh

```diff
@@ -21,6 +19,19 @@ debug_mode
 # Must be root
 root_check
 
+# Check if fulltextsearch is already installed
+print_text_in_color "$ICyan" "Checking if Fulltextsearch is already installed..."
+if does_this_docker_exist "$nc_fts"
+then
+    msg_box "It seems like 'Fulltextsearch' is already installed.\nIf you continue, Fulltextsearch will get reinstalled."
+    if [[ "no" == $(ask_yes_or_no "Do you really want to continue?") ]]
+    then
+        exit
+    fi
+fi
+
+print_text_in_color "$ICyan" "(Re-)Installing Elastic Search & Full Text Search on Nextcloud..."
```

Comment:
Move reinstalling to the if statment and else Installing otherwise it will say Re-installing when it's the first time you install.

Reply:
this should be done 👍